testing/upload_validation.ts

```diff
@@ -430,16 +445,6 @@ const commonPackFormulaSchema = {
   extraOAuthScopes: z.array(z.string()).optional(),
 };
 
-const numericPackFormulaSchema = zodCompleteObject<Omit<NumericPackFormula<any>, 'execute'>>({
-  ...commonPackFormulaSchema,
-  resultType: zodDiscriminant(Type.number),
-  schema: zodCompleteObject<NumberSchema>({
-    type: zodDiscriminant(ValueType.Number),
-    codaType: z.enum([...NumberHintValueTypes]).optional(),
-    description: z.string().optional(),
-  }).optional(),
-});
-
 const stringPackFormulaSchema = zodCompleteObject<Omit<StringPackFormula<any>, 'execute'>>({
   ...commonPackFormulaSchema,
   resultType: zodDiscriminant(Type.string),
```

Comment:
I mean that line 451 probably needs to say `schema: stringPropertySchema.optional()`, in order for string formula return values to pick up all the new fields that you just added support for in object schema properties.

You correctly did this for `numericPackFormulaSchema`. On line 566, you changed the schema validator to `schema: numberPropertySchema.optional(),` so for the same reason, you need to make a similar change for string formulas, right?

Reply:
Ah right, sorry, keep losing track of some of these. Updated